checker/manual/creating-a-checker.tex

```diff
@@ -1094,31 +1087,84 @@ \section{The checker class:  Compiler interface\label{writing-compiler-interface
 checker classes for the Interning Checker and the Nullness Checker:
 
 \begin{Verbatim}
-  @TypeQualifiers({ Interned.class, PolyInterned.class, PolyAll.class })
   @SupportedLintOptions({"dotequals"})
   public final class InterningChecker extends BaseTypeChecker { }
 
-  @TypeQualifiers({ Nullable.class, Raw.class, NonNull.class, PolyNull.class, PolyAll.class })
   @SupportedLintOptions({"flow", "cast", "cast:redundant"})
   public class NullnessChecker extends BaseTypeChecker { }
 \end{Verbatim}
 
-
 The checker class must indicate the annotations
 that make up the type hierarchy for this checker (including
-polymorphic qualifiers), either via a
-\refqualclass{framework/qual}{TypeQualifiers} annotation or by overriding
+polymorphic qualifiers), either by placing the annotations inside a subfolder called \<qual> located in the same directory as the checker, or by overriding
```

Comment:
Please wrap any new text lines to 80 columns (here and elsewhere).  That is consistent with the file's current style.  It also makes diffs easier to read in the future.